### ISSUE TITLE
Downloaded subtitles don't shows

### ISSUE THREAD

**Issue by MateuszCz** [guancio/ChromeOsSubtitle#88]
Hello. I'm trying to watch something with subtitles but I can't. Movie goes with sound, file with subtitles is loaded but they won't shows.
![screenshot 2017-01-06 at 21 23 08](https://cloud.githubusercontent.com/assets/24965754/21732062/e2ba824e-d457-11e6-8593-8d0ede041a4e.png)
Am I doing something wrong and that's why i don't see them?
Best regards
Mat

**Comment by vivekannan** [comment]
Thank you for the screenshot! Can you also provide the name of the sub? There is probably a parse error or something like that. Hard to debug without the sub under question.

**Comment by MateuszCz** [comment]
Hello there!
Here it goes:
movie: Captain.America.Civil.War.2016.720p.BluRay.x264-[YTS.AG].mp4
sub: Captain.America.Civil.War.2016.720p.BluRay.x264-[YTS.AG].srt
at the movie starts I got info that 

> Captain.America.Civil.War.2016.720p.BluRay.x264-[YTS.AG].srt loaded

[Captain.America.Civil.War.2016.720p.BluRay.x264-[YTS.AG].zip](https://github.com/guancio/ChromeOsSubtitle/files/691752/Captain.America.Civil.War.2016.720p.BluRay.x264-.YTS.AG.zip)

**Comment by vivekannan** [comment]
Yep. Just as I suspected. The content of that file doesn't follow the SRT
format. Not sure which format it is. Try converting it to proper SRT. I
will include code to display an error is the file format is improper.

**Comment by vivekannan** [comment]
Also, you are one of the few who is responsive while making bug reports.
Would you be interested in beta testing the latest unreleased version?
